docs/pages/blog/mui-core-v5.md

```diff
@@ -492,11 +496,69 @@ The following components are now available in the lab:
 
 ## v4 migration
 
-- The high-level changes required
-- Installation
-- ⚓️ We have introduced a new release line: v4.x.x-deprecations.x. This release line is kept in sync with the latest version of v4 and includes actionable deprecations to ease the migration to v5.
-- The codemod, covered a bit in https://material-ui.com/blog/2021-q2-update/
-- The migration guide
+We put a lot of work in order to make the migration from v4 to v5 as easier as possible. If you are starting your upgrade, these are the three things you should look into:
+
+- ⚓ We have introduced a new release line: v4.x.x-deprecations.x. This release line is kept in sync with the latest version of v4 and includes actionable deprecations to ease the migration to v5.
+- ⚒️ We have prepared a [codemod](/guides/migration-v4/#preset-safe) that does most of the transformations you will need for the migration. If you are not familiar what a codemod is, check out [Effective Refactoring with Codemods by Edd Yerburgh](https://www.youtube.com/watch?v=H9qtLutnT_g&ab_channel=Pusher).
+- 📄 Lasty, we have prepared a step by step [guide](/guides/migration-v4/) about how you can upgrade to v5, using the codemod above. This guide is the one place where you can find all information required for upgrading to v5.
+
+In the following sections, we will cover some high-level changes required for a successfull upgrade.
+
+### Change of the package names
+
+In order to support the new branding and the long term direction we have, we made a change in the terminology used in the project.
+To support this, we needed to change the names of the packages that we provide. For more details on this check the [GitHub discussion](https://github.com/mui-org/material-ui/discussions/27803).
+
+### Change of the styling engine
+
+The change of the styling engine, allowed us to unlock the improvement of the DX for the customization of the components, and the performance of the dynamic styles.
+We replaced [JSS](https://cssinjs.org/?v=v10.7.1) with [emotion](https://emotion.sh/docs/introduction) as a default styled engine.
+If you preffer [styled-components](https://styled-components.com/), see the next version of how you can use it instead of emotion.
+
+We recommend migration your `makeStyles` customization by using the new customization APIs: `styled` or the `sx` prop.
+However, if you still wish to use the `makeStyles` API, you can:
+
+- add `@mui/styles` as a dependency and change the imports of the `makeStyles`/`withStyles` utilities
+- use [`tss-react`](https://github.com/garronej/tss-react) - it has API is similar to JSS makeStyles but works with emotion
+
+You can find more information for this on the [Migrate `makeStyles` to emotion](/guides/migration-v4/#migrate-makestyles-to-emotion) section of the migration guide.
```

Comment:
https://deploy-preview-27912--material-ui.netlify.app/guides/migration-v4/#migrate-from-jss

I have changed this header to be more accurate:

```suggestion
You can find more information for this on the [Migrate from JSS](/guides/migration-v4/#migrate-from-jss) section of the migration guide.
```